docs/api/browser-window.md

```diff
@@ -14,7 +14,11 @@ win.on('closed', () => {
   win = null
 })
 
+// Load a remote URL
 win.loadURL('https://github.com')
+
+// Or load a local HTML file
+win.loadURL(`file://${__dirname}/app/index.html`)
```

Comment:
I added the full-blown `url.format` example to the top of the `browser-window.md` example, but it seemed more distracting than useful. I also think the shorthand string format will work for the majority of users who aren't using unusual characters in their filenames.

I do agree the `url.format` technique is useful though, so I added a note about how to properly use it for `file` URLs within the description of the `loadURL` method.